### DISCUSSION TITLE
CODEOWNERS and their respective teams

### DISCUSSION THREAD

**Discussion by ovflowd** [nodejs/nodejs.org#5187] [Ideas]
Hey, y'all 👋 I wanted to update our CodeOwners to reflect the actual teams that can give insights about each aspect of the Website.

- I was thinking of adding the @nodejs/i18n group for changes to the `crowdin.yml` which is more of a Localisation config file. Would i18n be the right group for that as they/we manage Crowdin?
- Also, could the @nodejs/typescript team be the right one to give insights on better standards if we ever want to update the website types and its `tsconfig.json`?
- For Security Blog Posts and Announcements. Would it make sense for @nodejs/security to be added as CODEOWNERS?
- For the About and Get Involved sections, it also does make sense to have the @nodejs/tsc to be pinged as they own pretty much the content regarding "About Node.js" and "Get Involved".

What do y'all think? (I intentionally pinged these teams to get their opinions)

**Comment by nschonni** [comment]
Side note, for the reviews to work, the teams need to have write access to the repo, unless that's changed from before. That was why the old language specific teams entries in the file didn't work in the past (but I never cleaned them out)

**Reply by ovflowd** [reply]
Aware of that, I already gave access to all these teams when I updated the `CODEOWNERS` (I'm aware of how CODEOWNERS works 🙇). I also updated branch rules, that even by them having `write` access, they still require someone from Website to approve and merge.

So there are a bunch of rules in place to ensure that no accidental actions or bad-actors force merge things here.

**Reply by ovflowd** [reply]
> That was why the old language specific teams entries in the file didn't work in the past (but I never cleaned them out)

Oh yeah, I saw that!

**Reply by ovflowd** [reply]
Anyhow, thanks for the pro-tip, I appreciate it! Glad that we both noticed that 👀

**Reply by kyledrake6o9** [reply]
- - [ ] @nschonni @ovflowd @bmuenzenmeyer @nodejs-github-bot it's mine I'm
Taking it!

**Reply by ovflowd** [reply]
Sorry, what are you talking about?
